backend/src/main/java/meetup/connect/event/Event.java

```diff
@@ -0,0 +1,50 @@
+package meetup.connect.event;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+public class Event {
```

Comment:
praise: I'm a big fan of that file structure. It enables you to apply package-private access.